pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/sort/Alias.java

```diff
@@ -86,8 +89,10 @@ public static Alias of(SortBy sortBy) {
 
     public static Alias of(ParticipantColumn column) {
         Alias esAlias;
-        if (Objects.nonNull(column.getObject())) {
+        if (Objects.nonNull(column.getObject()) && Alias.ofOrNull(column.getObject()) != null) {
```

Comment:
I was thinking that this method is pretty related to `ParticipantColumn` class, I believe no one will create object of `ParticipantColumn` to get alias, everyone will use `private static Alias of(String alias)` in most of the cases, `public static Alias of(ParticipantColumn column)` is specific case only for export feature, maybe we could move that method to `ParticipantColumn` and I believe it will become more cleaner that you will directly call for example `getAlias` method on the instance of `ParticipantColumn`, instead of calling `Alias.of` and passing instance of `ParticipantColumn` to it?

Reply:
Those are really good points.  the downside of moving it is that it relies on a lot of currently private constructors, ad making those public would lose some of the nice encapsulation Alias gives now.  I think leaving it in will help us better to say "Alias" handles all logic related to mapping filters to tables.